instant.perms.ts

```diff
@@ -0,0 +1,17 @@
+// Docs: https://www.instantdb.com/docs/permissions
+
+import type { InstantRules } from "@instantdb/react-native";
+
+const rules = {
+  todos: {
```

Comment:
You may also want to add a $default: "false" at the top level, so users can't create other tables, etc